src/main/resources/schemas/schema-form.json

```diff
@@ -26,6 +26,16 @@
       "title": "URL",
       "type" : "string"
     },
+    "conTimeout": {
+      "title": "Connection Timeout (ms)",
+      "default": 1000,
+      "type" : "number"
```

Comment:
It should probably be an `integer` here.
Also, can you define the min value please.

```
"connectionTimeout" : {
          "type" : "integer",
          "title": "Connection Timeout (ms)",
          "description": "Maximum time to wait for before considering a connection timeout.",
          "minimum": 0
        },
```